src/script/components/blog-showcase.ts

```diff
@@ -356,15 +352,25 @@ export class ResourceHub extends LitElement {
           <img src="${post.imageUrl}" alt="${post.title} card header image" />
           <div class="card-content">
             <h2 class="title">${post.title}</h2>
-            <fast-button id="share" class="share" appearance="lightweight"
-              >Share</fast-button
+            <app-button id="share" class="share" appearance="lightweight"
+              >Share</app-button
             >
           </div>
         </fast-card>
       `;
     });
   }
 
+  renderBlogActions() {
+    if (window.innerWidth > BreakpointValues.smallUpper) {
+      return html`
+        <div id="blog-actions">
+          <app-button>${this.viewBlogButtonText()}</app-button>
+        </div>
+      `;
+    }
+  }
+
```

Comment:
Future note: Would it be cleaner to have classes for the breakpoints? e.g. hidden-xs, hidden-sm, etc.? Less JS logic and more CSS styles, which seems better in my mind's eye.

Reply:
yeah that's a good idea!